packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue

```diff
@@ -157,6 +168,32 @@ export default {
       type: [String, Number],
       default: 326
     },
+    /**
+     * Badge label
+     */
+    badgeLabel: {
+      type: String,
+      default: ""
+    },
+    /**
+     * Badge label
+     */
+    badgeType: {
+      type: String,
+      default: "warning",
+      validator: function(value) {
+        return ["secondary", "info", "success", "warning", "danger"].includes(
```

Comment:
how about calling it `badgeColor` and using same css modifiers as in SfIcon?

Reply:
Ok, sounds good to me